### PR TITLE
feat/bugs: Various bug fixes

### PR DESCRIPTION
**Pull request by Tulun**
**Motivation**
Address issues reported by last voting round.

**Summary**
Following tickets were addressed:

1) Nav bar should highlight the page the user is on. (https://app.clubhouse.io/uma-project/story/978/nav-bar-should-highlight-the-page-the-user-is-on)

2) When a tx is submitted the commit/reveal module does not show a link to the successful tx (note this will only work on Kovan or mainnet). (https://app.clubhouse.io/uma-project/story/824/when-a-tx-is-submitted-the-commit-reveal-module-does-not-show-a-link-to-the-successful-tx)

3) If you click "view details" on a "Past Request" the dapp crashes. This was a float issue with ethers Big number.
 (https://app.clubhouse.io/uma-project/story/973/if-you-click-view-details-on-a-past-request-the-dapp-crashes

4) Dropdown chevron should be under the selected dropdown not over (https://app.clubhouse.io/uma-project/story/979/dropdown-chevron-should-be-under-the-selected-dropdown-not-over)

5) I can’t see the “yes I’m ready” button unless i zoom out past 80%. We should make the module viewable at 100%. I adjusted the height of the portal to use percentage, not raw pixels. (https://app.clubhouse.io/uma-project/story/949/i-can-t-see-the-yes-i-m-ready-button-unless-i-zoom-out-past-80-we-should-make-the-module-viewable-at-100)

6) Fix table spacing (https://app.clubhouse.io/uma-project/story/981/fix-table-spacing)

7) Incomplete past requests displaying. This was the result of the gql query not only returning 100 values. Set to 1000. (https://app.clubhouse.io/uma-project/story/988/incomplete-past-requests-displaying)

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

**Review by daywiss — APPROVED** [review]
looks good as far as i can tell. went through each issue and QA'd. 
some comments on the UX:
1.  past request details show description but not title for admin votes eg: "Admin 98" ( this shows in active requests):
image

File: src/common/components/text-input/TextInput.tsx [hunk 1]
```diff
@@ -27,7 +36,7 @@ const _TextInput: ForwardRefRenderFunction<HTMLInputElement, Props> = (
           placeholder={props.placeholder}
           ref={field.ref}
           value={field.value}
-          onChange={field.onChange}
+          onChange={props.onChange ? props.onChange : field.onChange}
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/common/components/text-input/TextInput.tsx:39]: Sorry, this is for a different ticket -- trying to decide how to handle copy pasting data into the input and how we handle characters we don't like.

File: src/web3/get/queryRetrieveRewards.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export const queryRetrieveRewards = async (
 
     return ethers.utils.formatEther(reward.toString());
   } catch (err) {
-    console.log("err", err);
+    // We would expect an error here if the reward is available but not unclaimed.
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/web3/get/queryRetrieveRewards.ts:36]: yeah I'll think about this. As we actually expect errors sometimes on this call.

File: src/common/components/text-input/TextInput.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gamaranto) [reply src/common/components/text-input/TextInput.tsx:39]: nit, you can use `??` and get rid of the ternary

File: src/web3/get/queryRetrieveRewards.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gamaranto) [reply src/web3/get/queryRetrieveRewards.ts:36]: I think in general we want to catch errors and not bubble them up only when we can do something about them

File: src/features/vote/PastRequests.tsx [hunk 3]
```diff
@@ -65,6 +65,8 @@ const PastRequests: FC<Props> = ({
         (el) => Number(el.roundId) < Number(roundId)
       );
 
+      // console.log("FBR", filteredByRound, "raw data", voteSummaryData);
```
[the remark this answers is not in the document]
Comment (Gamaranto) [reply src/features/vote/PastRequests.tsx]: I think CRA does this by default

**Comment by Gamaranto** [comment]
Nit, and prob not related to this PR, but the border on the table rows can break down on smaller screen size
image

**Review by Gamaranto — COMMENTED** [review]
The tickets are resolved, just some minor things in code to adjust. Good job!

File: public/index.html [hunk 4]
```diff
@@ -6,6 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="UMA Voter dApp" />
+    <meta property="og:title" content="UMA Voter dApp" />
+    <meta
+      property="og:description"
+      content="UMA's Voting application for governance and price requests."
+    />
+    <meta property="og:image" content="%PUBLIC_URL%/favicon.ico" />
```
[the remark this answers is not in the document]
Comment (Tulun) [reply public/index.html:14]: Yep

File: src/hooks/useEncryptedVotesEvents.ts [hunk 5]
```diff
@@ -33,7 +33,11 @@ export default function useEncryptedVotesEvents(
     },
     {
       // do not run query if any of these are null
-      enabled: contract !== null && privateKey !== "" && address != null,
+      enabled:
+        contract !== null &&
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/hooks/useEncryptedVotesEvents.ts:37]: It's the context type, which is null or contract.ethers.

File: src/features/vote/PastViewDetailsModal.tsx [hunk 6]
```diff
@@ -65,7 +66,8 @@ const _PastViewDetailsModal: ForwardRefRenderFunction<
   // Format rewards to 6 decs. It is a Big Num as it the value is in wei.
   useEffect(() => {
     if (rewardsClaimed !== "-" && rewardsClaimed !== "0") {
-      const formatRCArr = ethers.utils.formatEther(rewardsClaimed).split(".");
+      const rcToWei = toWeiSafe(rewardsClaimed).toString();
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/features/vote/PastViewDetailsModal.tsx]: Fair!

File: src/features/vote/ActiveRequests.tsx [hunk 7]
```diff
@@ -26,6 +26,7 @@ export interface ModalState {
 interface Props {
   activeRequests: PendingRequest[];
   roundId: string;
+  refetchRoundId: Function;
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/features/vote/ActiveRequests.tsx]: Haha we can, although it's probably not the most helpful.

(
    options?: RefetchOptions | undefined
  ) => Promise<QueryObserverResult<string | void | undefined, unknown>>;

**Pull request merged**